spec/features/profiles_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+describe "Visiting profiles" do
+  include TestFactories
+
+  before do
+    @user = authenticated_user
+    @post = associated_post(user: @user)
+    @comment = Comment.new(user: @user, body: "A Comment")
+    allow(@comment).to receive(:send_favorite_emails)
+    @comment.save
+
+#    Capybara.current_driver = :webkit
```

Comment:
In case you've not encountered it yet, you can just flag a `describe` or `it` block with `js: true` to have Capybara switch to the default js driver for only that block. This means the spec won't need to call `Capybara.use_default_driver` or `Capybara.current_driver`.

For examples and explanation see the [Capybara README](https://github.com/jnicklas/capybara) and in-page search for "`:js`" to find the relevant docs.